docs/tutorials/09_Protein_Folding.ipynb

```diff
@@ -0,0 +1,708 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "739993c7",
+   "metadata": {},
+   "source": [
+    "# Protein Folding"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d64f8162",
+   "metadata": {},
+   "source": [
+    "### Introduction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3bd209d1",
+   "metadata": {},
+   "source": [
+    "The structure and function of many natural and human-engineered\n",
+    "proteins is still only poorly understood. As a result, our understanding of\n",
+    "processes connected with protein folding, such as those encountered in\n",
+    "Alzheimer’s disease, vaccine development, and crop improvement\n",
+    "research, has remained limited.\n",
+    "\n",
+    "Unfolded polypeptides have a very large number of degrees of freedom\n",
+    "and thus an enormous number of potential conformations. For example, a\n",
+    "chain with 100 amino acids has on the order of 10 conformations. In\n",
+    "reality, however, many proteins fold to their native structure within\n",
+    "seconds. This is known as Levinthal’s paradox [1].\n",
+    "\n",
+    "The exponential growth of potential conformations with chain length\n",
+    "makes the problem intractable for classical computers. In the quantum\n",
+    "framework, our resource-efficient algorithm scales linearly with\n",
+    "the number of amino acids N.\n",
+    "\n",
+    "\n",
+    "<font size='2'>[1] https://en.wikipedia.org/wiki/Levinthal%27s_paradox </font>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3110941f",
+   "metadata": {},
+   "source": [
+    "The goal of this work is to determine the minimum energy configuration of a protein. Starting from a random configuration, the protein's structure is optimized to lower the energy. This can be achieved by encoding the protein folding problem into a qubit operator and ensuring that all physical constrains are satisfied. "
```

Comment:
> configuration

You are mixing the use of `conformation` and `configuration`. But these are not interchangeable!

Reply:
Thanks!